packages/database/src/modelClasses/DataServiceSyncGroup.js

```diff
@@ -22,6 +22,10 @@ const syncStatuses = {
 class DataServiceSyncGroupType extends DatabaseType {
   static databaseType = TYPES.DATA_SERVICE_SYNC_GROUP;
 
+  async setSyncIdle() {
+    return this.model.update({ id: this.id }, { sync_status: syncStatuses.idle });
+  }
```

Comment:
Technically this is doing the exact same thing as `setSyncCompletedSuccessfully()`, however I kind of like having two separate methods as it's a bit of an implementation detail that they both just set the state to `IDLE`. Also it makes the code read a lot nicer 😄